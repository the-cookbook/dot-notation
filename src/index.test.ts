import dot from './';

describe('dot', () => {
  it('should match snapshot', () => {
    expect(dot).toMatchSnapshot();
  });
});
