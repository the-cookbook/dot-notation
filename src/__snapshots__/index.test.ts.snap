// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dot should match snapshot 1`] = `undefined`;
